modules/pensions/app/models/pensions/saved_claim.rb

```diff
@@ -85,6 +86,35 @@ def send_confirmation_email
           'confirmation_number' => guid
         }
       )
+
+      insert_notification(saved_claim_id)
+    end
+
+    ##
+    # insert notifcation post VANotify email send
+    #
+    # @param id [Pensions::SavedClaim]
+    # @see ClaimVANotification
+    #
+    def insert_notification(saved_claim_id)
+      ClaimVANotification.create!(
+        form_type: form_id,
+        saved_claim_id: saved_claim_id,
+        email_sent: true,
+        email_template_id: Settings.vanotify.services.va_gov.template_id.form527ez_confirmation_email,
+        created_at: DateTime.now,
+        updated_at: DateTime.now
+      )
+    end
+
+    ##
+    # Find notifcation
+    #
+    # @param id [Pensions::SavedClaim]
+    # @see ClaimVANotification
+    #
+    def find_notification(saved_claim_id)
+      ClaimVANotification.find_by(saved_claim_id:)
```

Comment:
since this table may/will be used for multiple templates, would need to also `find_by` email_template_id (which would be passed as a param). And for safety sake, also check for the form_type
change name to `has_notification?`?